### PR TITLE
Add licenses and sources to many projects

### PR DESCRIPTION
**Pull request by adelq**
Many projects did not have licenses or sources in data.json, many more to go.

### CODE REVIEW

**Comment by ironbishop** [comment]
:+1:; links verified, licenses verified; thanks @Adelq :)

Notes: 
- [Maqetta](https://github.com/maqetta/maqetta/blob/master/LICENSE) here is listed as "bsd", but in particular _"is available under either the terms of the modified BSD license or the Academic Free License version 2.1."_.
- [Beex](https://github.com/beexsvn/Beex.org/blob/master/license.txt) license is not listed, seems to be some sort of modified BSD...
- ofuz: TL,DFL (too long, didn't find license file)

@jancborchardt what do you think?

**Comment by yuwash** [comment]
Checked up to limesurvey (l. 108). → :+1: Seems good. I just can't find where the license of [SAGE](http://hg.sagemath.org/) (l. 102, GNU GPL 2).
- Beex license ([license.txt](https://github.com/beexsvn/Beex.org/blob/master/license.txt)) is not described as BSD, so I think we shouldn't describe it as such
- ofuz license is clearly noted in its [README](https://github.com/philfree/ofuz/blob/master/README)

**Comment by ironbishop** [comment]
sage license file is [here](http://hg.sagemath.org/sage-root/src/24191e75e4e98bcbbd8183df2458ea5f5dbd6567/COPYING.txt?at=default) ...but it's a bit complicated:boom:

**Comment by yuwash** [comment]
Thanks @ironbishop for finding that. According to that, it's GPL3 instead of 2, so we need to change it.
Now I :sweat_smile: finished checking.
- backchan.nl (l. 111) is not BSD but a [custom license](https://github.com/drewww/backchan.nl/blob/master/LICENSE)
- the source link of codingteam (l.135) should be http://codingteam.net/project/codingteam
- net2ftp (l.139) is complicated :boom: as well. Additionally, there's no open development platform where you can browse the code online or join as contributor
- showslow (l.145) is not bsd but a [custom license](https://github.com/sergeychernyshev/showslow/blob/master/LICENSE)

**Comment by adelq** [comment]
Agreed on all points, but just some additional pointers:
- I did see maqetta being dual licensed under AFL, but since AFL wasn't defined I kept it to BSD. I'll add a commit to fix that. 
- [net2ftp](http://www.net2ftp.com/LICENSE.txt) looks like it is licensed as GPLv2, but many of the included utilities are not
- backchan.nl appears to be a [bsd 3-clause license](http://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_.28.22Revised_BSD_License.22.2C_.22New_BSD_License.22.2C_or_.22Modified_BSD_License.22.29) although it is not labeled as such
- showslow also appears to be a [bsd 3-clause license](http://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_.28.22Revised_BSD_License.22.2C_.22New_BSD_License.22.2C_or_.22Modified_BSD_License.22.29) but is not labeled
- Beex does appear to be a modified BSD license, so it was left unlabeled
- The [sage](http://hg.sagemath.org/sage-root/src/24191e75e4e98bcbbd8183df2458ea5f5dbd6567/COPYING.txt#cl-104) license labels sage itself as GPLv2+, however some of the addons are not

**Comment by ironbishop** [comment]
Backchan, net2ftp, showslow: I agree with adelq.
Magetta: that was only a note, BSD it's fine for me. In case of dual licensing, we can list the most permissive.

Sage:
- line 8 " licensed under the General Public License, version 3 (no other versions!)"
- line104: "GPLv2+"

Since the "+" in "GPLv2+" means that's [compatible](http://www.gnu.org/licenses/quick-guide-gplv3.html), I'd list Sage as GPLv3...

**Comment by yuwash** [comment]
Thanks adelq for responding on all the points.
- Agree that the license of backchannl and showslow is the BSD 3-clause. What do other @libreprojects/contributors say about labeling them as bsd?
- Sage: maybe gpl2 and gpl3 can be listed both if you prefer (if not I prefer gpl3 as well)

Beyond this, I'd say :+1: for this merge.

File: js/data.json [hunk 1]
```diff
@@ -19,6 +19,7 @@ var lp = {
 		{"id":"zpl-v2","name":"The Zope Public License v2","address":"http://opensource.org/licenses/zpl-2.0"},
 		{"id":"apache-v2","name":"Apache License v2","address":"https://www.apache.org/licenses/LICENSE-2.0.html"},
 		{"id":"epl-v1","name":"Eclipse Public License v1","address":"http://eclipse.org/org/documents/epl-v10.php"}
```
Comment (ironbishop) [inline js/data.json]: :warning: Missing comma at end of line. :warning:

Comment (adelq) [inline js/data.json]: My mistake, fix posted

**Comment by jancborchardt** [comment]
You are all awesome!

@adelq I added you to the team as well. In general though we always work with pull requests so that the others can review changes. :)

**Comment by adelq** [comment]
Thanks! :)

**Pull request merged**